test/utils-tests.js

```diff
@@ -349,4 +349,21 @@ describe('replaces all substrings', () => {
         utils.replaceAll("aaaaaaaa", "a", "ba")
             .should.equal("babababababababa");
     });
-});
\ No newline at end of file
+});
+
+describe('encodes in our version of base32', () => {
+    function doTest(original, expected) {
+        utils.base32Encode(Buffer.from(original)).should.equal(expected);
+    }
+
+   it('returns what we expect', () => {
```

Comment:
Do these test the case where we need the "last pass" ?

Reply:
Yes! But I just made it more explicit on the tests now